README.md

```diff
@@ -13,118 +13,14 @@ This is the home of [ar.io] SDK. This SDK provides functionality for interacting
-    - [`getAllGatewayVaults({ cursor, limit, sortBy, sortOrder })`](#getallgatewayvaults-cursor-limit-sortby-sortorder-)
-    - [`increaseOperatorStake({ qty })`](#increaseoperatorstake-qty-)
-    - [`decreaseOperatorStake({ qty })`](#decreaseoperatorstake-qty-)
-    - [`redelegateStake({ target, source, stakeQty, vaultId })`](#redelegatestake-target-source-stakeqty-vaultid-)
-    - [`getRedelegationFee({ address })`](#getredelegationfee-address-)
-    - [`getAllDelegates({ cursor, limit, sortBy, sortOrder })`](#getalldelegates-cursor-limit-sortby-sortorder-)
-  - [Arweave Name System (ArNS)](#arweave-name-system-arns)
-    - [`buyRecord({ name, type, years, processId })`](#buyrecord-name-type-years-processid-)
-    - [`getArNSRecord({ name })`](#getarnsrecord-name-)
-    - [`getArNSRecords({ cursor, limit, sortBy, sortOrder })`](#getarnsrecords-cursor-limit-sortby-sortorder-)
-    - [`increaseUndernameLimit({ name, qty })`](#increaseundernamelimit-name-qty-)
-    - [`extendLease({ name, years })`](#extendlease-name-years-)
-    - [`getTokenCost({ intent, ...args })`](#gettokencost-intent-args-)
-    - [`getCostDetails({ intent, fromAddress, fundFrom, ...args})`](#getcostdetails-intent-fromaddress-fundfrom-args)
-    - [`getDemandFactor()`](#getdemandfactor)
-    - [`getArNSReturnedNames({ cursor, limit, sortBy, sortOrder })`](#getarnsreturnednames-cursor-limit-sortby-sortorder-)
-    - [`getArNSReturnedName({ name })`](#getarnsreturnedname-name-)
-  - [Epochs](#epochs)
-    - [`getCurrentEpoch()`](#getcurrentepoch)
-    - [`getEpoch({ epochIndex })`](#getepoch-epochindex-)
-    - [`getEligibleEpochRewards({ epochIndex }, { cursor, limit, sortBy, sortOrder })](#geteligibleepochrewards-epochindex---cursor-limit-sortby-sortorder-)
-    - [`getObservations({ epochIndex })`](#getobservations-epochindex-)
-    - [`getDistributions({ epochIndex })`](#getdistributions-epochindex-)
-    - [`saveObservations({ reportTxId, failedGateways })`](#saveobservations-reporttxid-failedgateways-)
-    - [`getPrescribedObservers({ epochIndex })`](#getprescribedobservers-epochindex-)
-  - [Primary Names](#primary-names)
-    - [`getPrimaryNames({ cursor, limit, sortBy, sortOrder })`](#getprimarynames-cursor-limit-sortby-sortorder-)
-    - [`getPrimaryName({ name, address })`](#getprimaryname-name-address-)
-    - [`requestPrimaryName({ name })`](#requestprimaryname-name-)
-    - [`getPrimaryNameRequest({ initiator })`](#getprimarynamerequest-initiator-)
-  - [Configuration](#configuration)
+- [Networks (Mainnet, Testnet, etc.)](#networks-mainnet-testnet-etc)
+- [ARIO](#ario)
 - [Arweave Name Tokens (ANT's)](#arweave-name-tokens-ants)
-  - [ANT APIs](#ant-apis)
-    - [`init({ processId, signer })`](#init-processid-signer-)
-    - [`getInfo()`](#getinfo-1)
-    - [`getHandlers()`](#gethandlers)
-    - [`getState()`](#getstate)
-    - [`getOwner()`](#getowner)
-    - [`getControllers()`](#getcontrollers)
-    - [`getRecords()`](#getrecords)
```

Comment:
where did these go?

Reply:
I set max depths in the doc generator to 2. The TOC was getting to large. The links are persisted, just not reference directly in the TOC.